src/main/scala/wdlTools/syntax/package.scala

```diff
@@ -64,8 +64,8 @@ object TextSource {
 
 // Syntax error exception
 final class SyntaxException(message: String) extends Exception(message) {
-  def this(msg: String, text: TextSource) =
-    this(s"${msg} at ${text}")
+  def this(msg: String, text: TextSource, docSourceURL: Option[URL] = None) =
+    this(s"${msg} at ${text}${docSourceURL.map(url => s" in ${url.toString}").getOrElse("")}")
```

Comment:
Yes, it is fun to write this in one line. However, for readability, can you please break this into two lines?

Reply:
I tried but it doesn't seem to work - the compiler complains about "this does not accept arguments"

Reply:
It looks like Java requires that if you make a call to this() in the constructor it has to be the first line.

Reply:
I added a separate formatMessage function that I call from the constructor.